### ISSUE TITLE
[Question] Does fsrs deals well with cramming

### ISSUE THREAD

**Issue by ganqqwerty** [open-spaced-repetition/fsrs4anki#640]
Do you have a doubt about using FSRS in Anki? If so, please post on https://forums.ankiweb.net/ instead.

This issue tracker is primarily intended for questions and suggestions about the algorithm itself, rather than its usage in Anki. In addition, a larger community can provide support on the Anki Forums.

Before posting, please read the following:

**Research**

*Enter an [x] character to confirm the points below:*

- [x] I have checked the [FAQ](https://github.com/open-spaced-repetition/fsrs4anki/blob/main/docs/tutorial.md#faq) and could not find an answer to my question
- [x] I have read the [wiki](https://github.com/open-spaced-repetition/fsrs4anki/wiki) and still felt confused
- [x] I have searched for similar existing questions here

**Question**

Sometimes I feel really energetic and want to do additional work in anki. I especially like increasing my retention by cramming sessions in anki. Does fsrs deals well with that activity?

**Comment by L-M-Sherlock** [comment]
Yeah. FSRS could deal with ahead reviews and delayed reviews by its DSR model.
